infra/terraform/test-org/org/locals.tf

```diff
@@ -242,6 +242,7 @@ locals {
       description = "Creates opinionated BigQuery datasets and tables"
       topics      = local.common_topics.da
       owners      = ["davenportjw", "shanecglass"]
+      groups      = [local.jss_common_group]
```

Comment:
Hi @NimJay - This `terraform-google-modules` org repo, so will need to use a `terraform-google-modules` org team.

Reply:
@NimJay - do you still need this onboarded? If so, can you make the suggested change?

Reply:
Thanks for following up, you two.

1. I just created b/337290831. @apeabody, when you find time, could you help with that request — I need an admin from terraform-google-modules?
2. We can keep value set to `local.jss_common_group` since both orgs (GoogleCloudPlatform and terraform-google-modules) will use the same team name: [`jump-start-solutions-admins`](https://github.com/orgs/GoogleCloudPlatform/teams/jump-start-solutions-admins).

Reply:
https://github.com/orgs/terraform-google-modules/teams/jump-start-solutions-admins has been created.